usecases/recording.cameras.opencv.3/README.md

```diff
@@ -0,0 +1,72 @@
+This folder provides the instructions for testing proxy-camera, which can be executed from a docker-compose file to start all micro-services to record video streams from three OpenCV cameras with lossless H264 compression. It is assumed that git, Docker, and Docker Compose are installed and all the three cameras are properly connected. To install Docker, follow the tutorial: https://docs.docker.com/engine/installation/linux/ubuntulinux/
+
+### Prepare OpenDaVINCI Docker image
```

Comment:
@HangYin2015 Lines 3--14 can be removed as this is part of the Dockerfile already.